### ISSUE TITLE
Unable to change app icons

### ISSUE THREAD

**Issue by BenHamler** [threethan/LightningLauncher#112]
Unsure if this is just me or I'm doing something wrong. I am attempting to change some blurry app icons to some custom images that I have transferred to my device via SideQuest. When I click on the app icon to change it, it launches a file explorer with a "screenshots" folder but I can't navigate anywhere else inside the file explorer. If I click the back arrow it kicks me back to the Lightning Launcher home screen. I have tried placing the images in the screenshots folder but they don't seem to appear on the device despite being in .jpg format. Is there a specific screenshots folder that Lightning Launcher is using? I have tried rebooting my device as well as re-installing Lightning Launcher but this issue seems to persist.

**Comment by threethan** [comment]
This isn't a proper file explorer, but an image browser. You should see any folders in your device's publicly-accessible storage which contains images. (For you, this seems to currently be only your screenshots folder) Placing a file in that folder should work. The image viewer is currently provided by a rather old library which may have issues and may get replaced in the near future.

Also, which app icons appear blurry? All non-sideloaded apps should now receive correct, high-res icons automatically.

**Comment by threethan** [comment]
I pushed a release that uses AndroidX for the image picker. See if it works any better for you.

**Comment by BenHamler** [comment]
The new update appears to have fixed it! I was able to navigate to the location I had stored the images on my Quest 3 device and update them successfully. The apps that were appearing with low resolution icons were: First Encounters, Les Mills Body Combat, Drop Dead The Cabin, and Beat Saber. Oddly enough, the new version automatically fixed the Beat Saber icon but I still had to manually update the others.
